src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -261,6 +309,50 @@ public T ToScalar()
             return Unsafe.As<Vector128<T>, T>(ref Unsafe.AsRef(in this));
         }
 
+        /// <summary>Converts the current instance to an equivalent string representation.</summary>
+        /// <returns>An equivalent string representation of the current instance.</returns>
+        /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
+        public override string ToString()
+        {
+            return ToString("G");
+        }
+
+        /// <summary>Converts the current instance to an equivalent string representation using the specified format.</summary>
+        /// <param name="format">The format specifier used to format the individual elements of the current instance.</param>
+        /// <returns>An equivalent string representation of the current instance.</returns>
+        /// <exception cref="NotSupportedException">The type of the current instance (<typeparamref name="T" />) is not supported.</exception>
+        public string ToString(string format)
+        {
+            return ToString(format, CultureInfo.CurrentCulture);
```

Comment:
It _may_ make sense to pass `null` for the `formatProvider`. That way `float` and `double` don't need to turn this CultureInfo into a NumberFormatInfo.

It even has a comment as the "fast path" in [https://github.com/dotnet/coreclr/blob/5feb0389f1e1ba715fba5560c979ac6362b1bcf8/src/System.Private.CoreLib/shared/System/Globalization/NumberFormatInfo.cs#L221-L222](https://github.com/dotnet/coreclr/blob/5feb0389f1e1ba715fba5560c979ac6362b1bcf8/src/System.Private.CoreLib/shared/System/Globalization/NumberFormatInfo.cs#L221-L222)

Reply:
Will update, it looks like all the supported primitive types pass `null` as well.